docs/standard/library-guidance/index.md

```diff
@@ -23,18 +23,20 @@ Aspects of high-quality open-source .NET libraries:
 
 ## Recommendations
 
-With each article, there is a list of recommendations for your open-source .NET library using **Do**, **Consider**, **Avoid**, and **Do not**. The wording of each recommendation indicates how strongly it should be followed. For example, a **Do** recommendation is one that should almost always be followed:
+With each article, there is a list of recommendations for your .NET library using **Do**, **Consider**, **Avoid**, and **Do not**. The wording of each recommendation indicates how strongly it should be followed.
+
+A **Do** recommendation is one that should almost always be followed:
 
 **✔️ DO** distribute your library using a NuGet package.
 
 On the other hand, **Consider** recommendations should generally be followed, but there are legitimate exceptions to the rule and you shouldn't feel bad about not following the guidance:
 
 **✔️ CONSIDER** using [SemVer 2.0.0](https://semver.org/) to version your NuGet package.
 
-**Do not** indicates something you should almost never do:
+**Avoid** recommendations are things that aren't a good idea, but breaking the rule sometimes makes sense:
 
-**❌ DO NOT** publish strong-named and non-strong-named versions of your library. For example, `Contoso.Api` and `Contoso.Api.StrongNamed`.
+**❌ AVOID** NuGet package references that demand an exact version.
 
-And finally, less strong, **avoid** recommendations are something isn't a good idea, but breaking the rule sometimes makes sense:
+And finally, **Do not** indicates something you should almost never do:
```

Comment:
nit: Do not -> do not

Reply:
If this should be "do not", should the "Do" be a "do"?